### PR TITLE
Add DLC jupyter notebook for model inference processing, and incorporate photometry pipeline redesign

### PR DESCRIPTION
**Pull request by sidhulyalkar**
- Modified Requirements to handle new scanreader installation for element calcium imaging
- Correct mistyped units in reference.py
- Add key source for ingestion.py, to prevent sessions without session_directory specified to be tried

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,11 +9,13 @@ element-event @ git+https://github.com/datajoint/element-event.git
...
+scanreader @ git+https://github.com/atlab/scanreader.git
+suite2p @ git+https://github.com/datajoint-company/suite2p.git
 graphviz
 tdt==0.4.3
 tifffile==2021.11.2
 nd2==0.1.6
-seaborn==0.12.0
\ No newline at end of file
+seaborn==0.12.0
+plotly==5.7.0
```
Comment (JaerongA) [inline requirements.txt]: Is plotly needed? I don't see it being used anywhere..  Also, why do we pin to this version?

File: workflow/pipeline/dlc.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
 from element_deeplabcut import train, model
-
 from workflow import db_prefix
 from workflow.utils.paths import get_dlc_root_data_dir, get_dlc_processed_data_dir
-
 from workflow.pipeline import lab, session, reference
+from workflow.pipeline.dlc import ModelVideoSet
```
Comment (JaerongA) [inline workflow/pipeline/dlc.py]: why is this being imported?

File: workflow/pipeline/dlc.py [hunk 3]
```diff
@@ -1,9 +1,12 @@
 from element_deeplabcut import train, model
-
 from workflow import db_prefix
 from workflow.utils.paths import get_dlc_root_data_dir, get_dlc_processed_data_dir
```
Comment (JaerongA) [inline workflow/pipeline/dlc.py:3]:
```suggestion

```

unused

File: workflow/pipeline/dlc.py [hunk 4]
```diff
@@ -13,3 +16,70 @@
...
+        else model.get_dlc_root_data_dir()
+    )
+    dlc_config["project_path"] = (
+        root_data_dir / "dlc_projects" / str(config_file_path).split("/")[-2]
+    ).as_posix()
+
+    sample_paths = [f for f in project_path.rglob('dlc-models/*trainset*shuffle*')]
+    iterations_dir = project_path / 'dlc-models' 
+
+    iteration_list = [iteration for iteration in iterations_dir.glob('*') if iteration.is_dir()]
```
Comment (JaerongA) [inline workflow/pipeline/dlc.py]:
```suggestion

```

not used

File: requirements.txt (hunk 1 above)
Comment (sidhulyalkar) [reply requirements.txt]: Ah yes I don't think plotly is needed, I will update and remove it

File: workflow/pipeline/dlc.py (hunk 2 above)
Comment (sidhulyalkar) [reply workflow/pipeline/dlc.py]: Also, not needed, will remove and update

File: workflow/pipeline/dlc.py (hunk 3 above)
Comment (sidhulyalkar) [reply workflow/pipeline/dlc.py:3]: I believe these are used during the activation of model and train schemas

File: workflow/pipeline/photometry.py [hunk 5]
```diff
@@ -419,6 +424,78 @@ def make(self, key):
             )
 
         self.SyncedTrace.insert(synced_trace_list)
+            try:  # if meta info exists populate here
```
Comment (ttngu207) [inline workflow/pipeline/photometry.py]: @sidhulyalkar what is this block here? Indentation error? Is it meant to be here?

Comment (sidhulyalkar) [reply workflow/pipeline/photometry.py]: Fixed it in latest commit, The try block was integrated into a loop going through all fiber_ids, which was lost in the conflict resolution. I added this back and formatted it according to @JaerongA's fork

**Pull request merged**